components/DataUploadDialog.tsx

```diff
@@ -91,6 +91,9 @@ const DataUploadDialog: FC<DataUploadDialogProps> = ({ disabled }) => {
       <FormWrapper action={processCsvUploadWrapped} submitButtonText="Upload">
         <DataUploadForm file={file} setFile={setFile} />
       </FormWrapper>
+      <Button color="teal" onClick={() => window.location.reload()}>
```

Comment:
This is interesting. I was going to suggest "Save", but that would be misleading for people who do _not_ press it, who may think that the data is _not_ loaded in that case.

Reply:
It's a little difficult as a situation, we don't want to force refreshing of the page so the user sees the message about successful/failed insertions and so the user can continue uploading data. However, the table itself won't update to include any new applications if the user clicks off and the page isn't refreshed.

Reply:
The styling of this is a bit off right now too but I plan on doing all the styling together later on.

Reply:
Fine by me, Sir